src/main/java/lotto/domain/Lotto.java

```diff
@@ -4,19 +4,27 @@
 import java.util.List;
 import java.util.Objects;
 import java.util.Set;
+import java.util.stream.Collectors;
 
 public class Lotto {
     public static final int DEFAULT_QUANTITY = 6;
     private final Set<LottoNumber> lottoNumbers;
 
-    public Lotto(List<Integer> numbers) {
+    public Lotto(List<LottoNumber> numbers) {
         this.lottoNumbers = new LinkedHashSet<>();
```

Comment:
컬렉션을 인자로 받는 생성자를 사용해보면 어떨까요?
```suggestion
        this.lottoNumbers = new LinkedHashSet<>(numbers);
```